swagger-spec/registrations/children_list.yaml

```diff
@@ -0,0 +1,203 @@
+# /registrations/{node_id}/children/
+get:
```

Comment:
"Nodes" should be changed to "registrations" everywhere in this file. When a project is registered, all of its components (child nodes) are registered as well, thus becoming child registrations of the registered project.